Detector/DetComponents/CMakeLists.txt

```diff
@@ -16,10 +16,16 @@ find_package(DD4hep COMPONENTS DDG4 DDSegmentation DDRec REQUIRED)
 
 find_package(Geant4)
 
+find_package(ACTS COMPONENTS Core TGeoPlugin DD4hepPlugin Examples)
+#workaround for the lone -lGeom flag exported by ACTS
+link_directories(${ROOT_INCLUDE_DIR}/../lib)
```

Comment:
This should not be needed with ROOT in the `LINK_LIBRARIES` and `GEOM` in the required components. So if you know where this comes from, please open a ticket.